### ISSUE TITLE
Comment inside an object literal treated as whitespace, triggering a false positive `standard/object-curly-even-spacing`

### ISSUE THREAD

**Issue by lolmaus** [xjamundx/eslint-plugin-standard#13]
Hey.

Got this code which I consider stylistically fine:

``` js
import {belongsTo/*, hasMany*/} from 'ember-data/relationships'
```

I can put the caret anywhere inside the comment and hit Ctrl+Shift+/ to have it uncommented. Very efficient.

The code triggers this error:

> Expected consistent spacing (`standard/object-curly-even-spacing`)

I believe it should not trigger this error, unless there's stray whitespace after `*/`.

Please fix. :bow:

**Comment by Krinkle** [comment]
Same for the following:

```js
      var intuition = lib.fetch({ url: intuitionLoadUrl, timeout: 7000 /* foo bar */ });
```

**Comment by xjamundx** [comment]
Thanks for these, I don't really intend to fix these as it looks like `standard` is going to pick some standard for curly braces and array brackets AFAIK

**Comment by feross** [comment]
@xjamundx Yes, that's the plan.

**Comment by feross** [comment]
Closing as this rule will be changed in a future version of `standard`, hopefully 11.x or 12.x.
